src/Fhir.Anonymizer.Core.UnitTests/AnonymizationVisitorTests.cs

```diff
@@ -0,0 +1,357 @@
+﻿using System.Collections.Generic;
```

Comment:
Do we put this file in folder "Visitors" to keep consistency?

Reply:
Agree, put it in the visitor folder.